src/main/java/com/unstoppabledomains/resolution/NamingService.java

```diff
@@ -0,0 +1,29 @@
+package com.unstoppabledomains.resolution;
+
+import org.web3j.protocol.Web3j;
+import org.web3j.tx.TransactionManager;
+import org.web3j.tx.ClientTransactionManager;
+
+import com.unstoppabledomains.exceptions.NamingServiceException;
+
+import org.web3j.ens.NameHash;
+
+abstract class NamingService extends Utilities {
+    public static Web3j web3;
+    public static TransactionManager transactionManager;
+
+    public NamingService(String name, Web3j web3, Boolean verbose) {
+        super(name, verbose);
+        NamingService.web3 = web3;
```

Comment:
This is a bad practice to setup static properties from instances. Static properties are used to configure all instances but not vice versa. It may result in a fact that making `new Resolution(...)` changes behavior of existing ones.